packages/webpack-config-terra/webpack.config.js

```diff
@@ -120,7 +121,6 @@ const defaultWebpackConfig = (env = {}, argv = {}, options = {}) => {
               loader: MiniCssExtractPlugin.loader,
               options: {
                 hmr: !production, // only enable hot module reloading in development
-                sourceMap: true,
```

Comment:
deleted this one because it's not even a real option for the `MiniCssExtractPlugin`.